internal/api/v1beta1/groupversion_info.go

```diff
@@ -33,4 +35,15 @@ var (
 
 	// AddToScheme adds the types in this group-version to the given scheme.
 	AddToScheme = SchemeBuilder.AddToScheme
+
+	// Group is the CRD group
+	Group = "theketch.io"
 )
+
+func init() {
+	if group := os.Getenv("CUSTOM_GROUP"); group != "" {
+		Group = group
```

Comment:
Just want to check, but should we confirm that the group pulled from the environment conforms to k8s standards?